### PR TITLE
Scale up learning rate according to num workers in Estimator imagenet models.

### PR DESCRIPTION
**Pull request by yuefengz**

### CODE REVIEW

File: official/resnet/cifar10_main.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def cifar10_model_fn(features, labels, mode, params):
   features = tf.reshape(features, [-1, HEIGHT, WIDTH, NUM_CHANNELS])
   # Learning rate schedule follows arXiv:1512.03385 for ResNet-56 and under.
   learning_rate_fn = resnet_run_loop.learning_rate_with_decay(
-      batch_size=params['batch_size'], batch_denom=128,
+      batch_size=params['batch_size'] * params['num_workers'], batch_denom=128,
```
Comment (dubey) [inline official/resnet/cifar10_main.py]: Would this trip up `cifar10_model_fn_helper` in cifar10_test.py?  Perhaps change to `(params['num_workers'] if 'num_workers' in params else 1)`.

File: official/resnet/imagenet_main.py [hunk 2]
```diff
@@ -318,7 +318,7 @@ def imagenet_model_fn(features, labels, mode, params):
     base_lr = .128
 
   learning_rate_fn = resnet_run_loop.learning_rate_with_decay(
-      batch_size=params['batch_size'], batch_denom=256,
+      batch_size=params['batch_size'] * params['num_workers'], batch_denom=256,
```
Comment (dubey) [inline official/resnet/imagenet_main.py]: Would this trip up imagenet_test.py?

File: official/resnet/cifar10_main.py (hunk 1 above)
Comment (yuefengz) [reply official/resnet/cifar10_main.py]: Done.

File: official/resnet/imagenet_main.py (hunk 2 above)
Comment (yuefengz) [reply official/resnet/imagenet_main.py]: I made num_workers optional now.
